lib/test/tasks/run_file_size_checks.rb

```diff
@@ -17,7 +17,7 @@ class Test::Tasks::RunFileSizeChecks < Pallets::Task
     'groceries*.css' => (108..118),
     'groceries*.js' => (441..451),
     'home*.css' => (0..10),
-    'home*.js' => (146..156),
+    'home*.js' => (239..249),
```

Comment:
This is a significant increase, but we will save more bytes than this by virtue of no longer loading the devicons font. However, loading JavaScript is probably at least somewhat more costly, byte for byte, than loading a font. Overall, though, I think that the user-experienced rendering performance will actually probably be improved?, because I think that the SVGs will now be stored directly in the pre-rendered HTML pages, which I think will allow a user to see the skill icons more quickly than if their browser also had to download a font from a separate server, first.